pkg/reconciler/autoscaling/hpa/config/testdata/config-autoscaler.yaml

```diff
@@ -0,0 +1,209 @@
+# Copyright 2024 The Knative Authors
```

Comment:
is this file the correct one or a left over from copying?

Reply:
Or is it needed?

Reply:
The new config store has two cms that is watching: a) the serving autoscaler one, b) the cm for this component.
So we need this in tests https://github.com/knative-extensions/autoscaler-keda/blob/231941697d82aaff0bceeea95f225b9b01dd917f/pkg/reconciler/autoscaling/hpa/config/store_test.go#L33.